src/Dropdown/Dropdown.test.jsx

```diff
@@ -22,7 +22,7 @@ const menuOpen = (isOpen, wrapper) => {
 
 describe('<Dropdown />', () => {
   describe('renders', () => {
-    const wrapper = shallow(<Dropdown {...props} />);
+    const wrapper = mount(<Dropdown {...props} />);
```

Comment:
why we have to replace `shallow` with `mount` here?

Reply:
`shallow` don't call `componentDidUpdate`

there is [difference](https://gist.github.com/fokusferit/e4558d384e4e9cab95d04e5f35d4f913)